text-generation-inference/tests/test_warmup.py

```diff
@@ -27,7 +24,10 @@ def test_warmup_jetstream_pytorch():
         model_path, revision="", max_batch_size=2, max_sequence_length=sequence_length
     )
     request = create_request(id=0, inputs=input_text, max_new_tokens=max_new_tokens, do_sample=False)
-    batch = Batch(id=0, requests=[request], size=1, max_tokens=sequence_length)
+    # The maximum sequence length of the model is set to 1000, but warmup will round that up to the next power of two
+    # in prefill (256).
+    max_tokens = 250
```

Comment:
Should it be 256?

Reply:
That's what it's explained in the comment: the warmup call will round up the max_tokens to 256 automatically. This is because the prefill will use buckets and pad the input to a power of two.

Reply:
I wasn't sure it was the intent of the test to make sure it bucketizes aha - sg!